packages/advisor-components/src/ContextWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { Router } from 'react-router-dom';
+import { History } from 'history';
+
+import { AdvisorProduct } from './types';
+
+export const translations: Record<string, Record<string, string>> = {
+  en: require('../../compiled-lang/en.json'),
+};
+
+export interface ContextWrapperProps {
+  /**
+   * routerProps - passed to Router from react-router-dom
+   */
+  routerProps?: {
+    history: History;
+  };
+  env: AdvisorProduct;
+  children: React.ReactNode;
+}
+
+export const ProductContext = React.createContext(AdvisorProduct.rhel);
+
+const ContextWrapper: React.FC<ContextWrapperProps> = ({ routerProps, env, children }) => {
+  const locale = navigator.language.slice(0, 2);
+
+  return routerProps ? (
+    <Router {...routerProps}>
+      <IntlProvider locale={locale} messages={translations[locale]}>
```

Comment:
@gkarat you should be able to pass strings from parent component, which is preferable. Your application knows which strings should be used where, if we were to pass messages from this repository to translators they wouldn't have full picture. Plus whenever you'd introduce new language we'd have to pass the request to translators as well. Hence would be better to pass the strings from your app, instead of having them in `messages` object. You can still parametrize those messages on the app side. All of the information is there, no need to bring the `IntlProvider` over.